syft/pkg/cataloger/catalog.go

```diff
@@ -23,6 +25,15 @@ type Monitor struct {
 	PackagesDiscovered progress.Monitorable // the number of packages discovered from all registered catalogers
 }
 
+// CatalogResult provides the result of running a single cataloger against source
+type CatalogResult struct {
```

Comment:
```suggestion
type catalogResult struct {
```
since this type isn't being used outside of this package we shouldn't export it